### DISCUSSION TITLE
Expecting value: line 1 column 1 (char 0)

### DISCUSSION THREAD

**Discussion by arjanMeerten** [deforum-art/sd-webui-text2video#67] [General]
I get this error when I click generate:

Pipeline setup
Exception occured
Expecting value: line 1 column 1 (char 0)

**Comment by kabachuha** [comment]
Please, attach the full log

**Comment by I-ShadowStar** [comment]
After fixing a different installation issue by downloading the config json file from the Huggingface depot, I am now also getting this error:

> ModelScope text2video extension for auto1111 webui
> Git commit: eed8524d (Mon Mar 20 15:00:12 2023)
> Starting text2video
> Pipeline setup
> Exception occured
> Expecting value: line 1 column 1 (char 0)

Here is my complete log:
 i   Settings file found, loading
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 →   Updating Settings File
 ✓   SUCCESS
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 i   Launcher Version 1.5.10
 i   Found a custom WebUI Config
 i   No Launcher launch options
 i   Python 3.10 found : [REDACTED]
 i   Python is in PATH
 i   Git found and already in PATH at C:\Program Files\Git\cmd\git.exe
 i   Automatic1111 SD WebUI found at C:\[REDACTED]\stable-diffusion-webui
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
  Opening A1111 WebUI Launcher
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 i   Arguments are now: --medvram --xformers --listen --port 6969 --api --opt-split-attention --enable-insecure-extension-access --gradio-auth [REDACTED] --no-half-vae --cloudflared
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Webui
 ✓   SUCCESS
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: a1111-sd-webui-locon
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: a1111-sd-webui-tagcomplete
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: a1111-stable-diffusion-webui-vram-estimator
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: gif2gif
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: novelai-2-local-prompt
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: openpose-editor
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: posex
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-dynamic-thresholding
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: SD-latent-mirroring
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-additional-networks
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-ar
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-aspect-ratio-helper
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (20/20), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 14 (delta 8), reused 6 (delta 2), pack-reused 0
Unpacking objects: 100% (14/14), 25.26 KiB | 275.00 KiB/s, done.
From https://github.com/thomasasfk/sd-webui-aspect-ratio-helper
   d320bd4..5f5a746  main       -> origin/main
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-controlnet
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-cutoff
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-depth-lib
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-llul
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-lora-block-weight
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-modelscope-text2video
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: sd-webui-tunnels
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: SDAtom-WebUi-client-queue-ext
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: Stable-Diffusion-Webui-Civitai-Helper
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-composable-lora
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-images-browser
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-model-toolkit
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-prompt-travel
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-rembg
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-state
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: stable-diffusion-webui-two-shot
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 ↺   Updating Extension: UnivAICharGen
 ✓   SUCCESS
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 i   Arguments are now: --medvram --xformers --listen --port 6969 --api --opt-split-attention --enable-insecure-extension-access --gradio-auth [REDACTED] --no-half-vae --cloudflared
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
  WEBUI LAUNCHING VIA EMS LAUNCHER, EXIT THIS WINDOW TO STOP THE WEBUI
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 !   Any error happening after 'commit hash : XXXX' is not related to the launcher. Please report them on Automatic1111's github instead :
 ☁   https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/new/choose
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Cancel
venv "C:\[REDACTED]\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Commit hash: a9fed7c364061ae6efb37f797b6b522cb3cf7aa2
Installing requirements for Web UI





Launching Web UI with arguments: --autolaunch --medvram --xformers --listen --port 6969 --api --opt-split-attention --enable-insecure-extension-access --gradio-auth [REDACTED] --no-half-vae --cloudflared
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: C:\[REDACTED]\stable-diffusion-webui\extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
Civitai Helper: No setting file, use default
Additional Network extension not installed, Only hijack built-in lora
LoCon Extension hijack built-in lora successfully
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████| 271/271 [00:00<00:00, 33967.56it/s]
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████| 271/271 [00:00<00:00, 34032.65it/s]
cloudflared detected, trying to connect...
 * Running on [REDACTED]
 * Traffic stats available on http://127.0.0.1:58523/metrics
Loading weights [c633845498] from C:\[REDACTED]\stable-diffusion-webui\models\Stable-diffusion\[!UmiAI][Ani]Macross_v2.safetensors
Creating model from config: C:\[REDACTED]\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: C:\[REDACTED]\stable-diffusion-webui\models\VAE\SD-default-vae.safetensors
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(924): [REDACTED]

Model loaded in 4.7s (create model: 0.4s, apply weights to model: 2.2s, apply half(): 0.7s, load VAE: 0.1s, load textual inversion embeddings: 1.2s).
Running on local URL:  [REDACTED]

To create a public link, set `share=True` in `launch()`.
Startup time: 37.9s (import gradio: 2.0s, import ldm: 1.0s, other imports: 2.0s, list extensions: 3.4s, load scripts: 5.5s, load SD checkpoint: 4.9s, create ui: 14.5s, gradio launch: 4.3s).
Consuming a byte in the end state
Consuming a byte in the end state
ModelScope text2video extension for auto1111 webui
Git commit: eed8524d (Mon Mar 20 15:00:12 2023)
Starting text2video
Pipeline setup
Exception occured
Expecting value: line 1 column 1 (char 0)

**Comment by I-ShadowStar** [comment]
This issue is being caused by the way that Huggingface is presenting the configuration.json file. Using "Save As" on the link gets you an HTML document.

I had to copy and paste the contents of the json file to correct this error.

**Comment by Pkhtjim** [comment]
> This issue is being caused by the way that Huggingface is presenting the configuration.json file. Using "Save As" on the link gets you an HTML document.
> 
> I had to copy and paste the contents of the json file to correct this error.

Ah so that's it. Same with a VoC based Auto1111 UI build. Both configuration and gitattributes were 59k. Had to download the raw link and the right ones are 2kb. Thanks a bunch. Now to do some memory management with an out of memory error.

**Comment by mischaschaub** [comment]
> > This issue is being caused by the way that Huggingface is presenting the configuration.json file. Using "Save As" on the link gets you an HTML document.
> > I had to copy and paste the contents of the json file to correct this error.
> 
> Ah so that's it. Same with a VoC based Auto1111 UI build. Both configuration and gitattributes were 59k. Had to download the raw link and the right ones are 2kb. Thanks a bunch. Now to do some memory management with an out of memory error.

Wow thanks, following your track, hope to hear from your memory management solution...

**Comment by mischaschaub** [comment]
Success, first video, wow: What worked for me with my 11GB 2080 TI in VoC Webui solution was:
1) Launching Web UI with arguments: --medvram --always-batch-cond-uncond --no-half --xformers --autolaunch
2) NOT to use the option "LOW VRAM VAE"

**Comment by Crimsonfart** [comment]
> VoC Webui

What is VoC Webui?

**Comment by Pkhtjim** [comment]
> > VoC Webui
> 
> What is VoC Webui?

Visions of Chaos, an AI suite for Windows.

**Comment by mischaschaub** [comment]
VoC  ist the very best open source patreon platform for all the ML and AI
stuff, brings it all together in one working system, saves my life daily!
Have a look: https://softology.pro/

Am Mo., 20. März 2023 um 22:44 Uhr schrieb Crimsonfart <
***@***.***>:

> VoC Webui
>
> What is VoC Webui?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/deforum-art/sd-webui-modelscope-text2video/issues/12#issuecomment-1476976699>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMF56BKRCEMPCSBJFOVY7G3W5DFURANCNFSM6AAAAAAWBFP7IE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


--
